data/virt_autotest/kubevirt_tests/kubevirt-tests.conf

```diff
@@ -0,0 +1,102 @@
+[vmi_lifecycle_test]
```

Comment:
@nanzhg  Would you please add copyright ,  summary and maintainer info to this file?

Reply:
Added.